.github/workflows/linter.yml

```diff
@@ -0,0 +1,62 @@
+###########################
+###########################
+## Linter GitHub Actions ##
+###########################
+###########################
+name: Lint Code Base
+
+#
+# Documentation:
+# https://help.github.com/en/articles/workflow-syntax-for-github-actions
+#
+# For the Web Almanac we have the following rule changes from the defaults:
+#  * Markdown - we're a bit more relaxed about this since authors may be less technical
+#  * Python - Only change from default is to set ignored-modules=flask, flask-talisman...etc.
+#  * CSS - Tweaked to ignore some Prince/Normalise exceptions and some whitespace exceptions
+#  * JS - default rules
+#  * JSON - default rules
+#  * YML - default rules
+# Exception config files are in the .github/linters directory
+
+#############################
+# Start the job on all push #
+#############################
+on:
+  push:
+    branches-ignore:
+      - 'main'
```

Comment:
Are we relying on contributors to ensure their code was linted in their branches and not running in in PRs?

Reply:
That's a bit sucky that the [example action config](https://github.com/github/super-linter#example-connecting-github-action-workflow) is for pushed. Changed it to run on pull requests instead.